code/data_structures/src/list/singly_linked_list/operations/find/search_element_python.py

```diff
@@ -0,0 +1,80 @@
+# Python Utility to add element at the end of the Singly Linked List
+# Part of Cosmos by OpenGenus Foundation
+
+
+class Node:
+    '''
+    A Node in a singly-linked list
+    Parameters
+    -------------------
+    data :
+        The data to be stored in the node
+    next:
+        The link to the next node in the singly-linked list
+    '''
+
+    def __init__(self, data=None, next=None):
+        ''' Initializes node structure'''
```

Comment:
I feel like the comments in both functions are somewhat unnecessary and even redundant.